drivers/SmartThings/zwave-switch/src/multichannel-device/init.lua

```diff
@@ -0,0 +1,93 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+local cc = require "st.zwave.CommandClass"
+local capabilities = require "st.capabilities"
+local st_device = require "st.device"
+local MultiChannel = (require "st.zwave.CommandClass.MultiChannel")({ version = 3 })
+
+local map_device_class_to_profile = {
+  [0x10] = "metering-switch",
+  [0x31] = "metering-switch",
+  [0x11] = "metering-dimmer",
+  [0x08] = "metering-sensor",
```

Comment:
Thermostat generic type maps to Metering sensor?

Reply:
Ok, I looked into the DTH again an it maps to Multi Sensor Endpoint, which contains temperature, luminance and humidity and Sensor Endpoints contains water, motion, smoke, water